assets/js/metrics_live/index.js

```diff
@@ -6,7 +6,13 @@ const SeriesValue = (options) => {
   if (!options.unit) return {}
 
   return {
-    value: (u, v) => v === null ? '' : v.toFixed(3) + ` ${options.unit}`
+    value: (u, v) => {
+      if (v === null || typeof v === 'undefined') {
```

Comment:
i'd still recommend `v == null` for this whole `if`

Reply:
Good point, thanks!